dashboard/app/controllers/level_starter_assets_controller.rb

```diff
@@ -102,7 +102,11 @@ def get_object(s3_filename)
   end
 
   private def file_mime_type(extension)
-    MIME::Types.type_for(extension)&.first&.raw_media_type
+    type = MIME::Types.type_for(extension)&.first
+    if type == MIME::Types['application/pdf']
+      return 'pdf'
+    end
```

Comment:
Note that this follows the same logic used by the `bucket_helper` in the files API, used when fetching student project assets:
https://github.com/code-dot-org/code-dot-org/blob/6f33d021e493ca95f586c91469669e25d016399a/dashboard/legacy/middleware/helpers/bucket_helper.rb#L39-L42